src/logic/documents/scorecards.js

```diff
@@ -63,6 +64,105 @@ export const downloadScorecards = (wcif, rounds, rooms, language) => {
   });
 };
 
+export const downloadEmptyScorecardsForPersons = (
+  wcif,
+  selectedPersons,
+  language = 'en'
+) => {
+  const personsWithoutGroups = competitorsRegisteredForAnEventWithoutGroups(
+    wcif
+  );
+  const filteredPersonsWithoutGroups = personsWithoutGroups.filter(
+    ({ person }) => selectedPersons.includes(person.wcaUserId)
+  );
+  const { scorecardsBackgroundUrl, scorecardPaperSize } = getExtensionData(
+    'CompetitionConfig',
+    wcif
+  );
+
+  getImageDataUrl(scorecardsBackgroundUrl).then(imageData => {
+    const pdfDefinition = scorecardsPdfDefinition(
+      emptyScorecardsForPersons(wcif, filteredPersonsWithoutGroups, language),
+      imageData,
+      scorecardPaperSize
+    );
+    pdfMake
+      .createPdf(pdfDefinition)
+      .download(`${wcif.id}-missing-scorecards.pdf`);
+  });
+};
+export const emptyScorecardsForPersons = (
+  wcif,
+  personsWithoutGroups,
+  language = 'en'
+) => {
+  const {
+    localNamesFirst,
+    printOneName,
+    printStations,
+    scorecardPaperSize,
+    scorecardOrder,
+  } = getExtensionData('CompetitionConfig', wcif);
+
+  const { scorecardsPerPage } = scorecardPaperSizeInfos[scorecardPaperSize];
+
+  let cards = flatMap(personsWithoutGroups, ({ person, eventIds }) =>
+    flatMap(eventIds, eventId => {
+      const round = wcif.events
+        .find(e => e.id === eventId)
+        ?.rounds?.find(r => r.id === `${eventId}-r1`);
+      const roundFormat = round?.format || '1';
+      const attemptCount = maxAttemptCountByFormat[roundFormat] || 5;
```

Comment:
We don't need to be defensive here, the round and format should be found, otherwise it means the data is corrupted, so it's better if this results in an error, rather than printing wrong scorecards.

```suggestion
      const round = wcif.events
        .find(e => e.id === eventId)
        .rounds.find(r => r.id === `${eventId}-r1`);
      const roundFormat = round.format;
      const attemptCount = maxAttemptCountByFormat[roundFormat];
```